cms/templates/home_page.html

```diff
@@ -10,9 +10,26 @@
 {% endblock %}
 
 {% block content %}
-  <h1>{{ page.title }}</h1>
-  <p>
-    Image:<br />
-    <img src="{% wagtail_img_src page.hero %}" alt="home" />
-  </p>
+  <style>
+    {% if page.hero %}
+      .hero-background {
+        background-image: url('{% wagtail_img_src page.hero%}');
+        background-size: cover;
+        background-position: 50% 50%;
```

Comment:
I think it would be best to move the `-size` and `-position` styles to a stylesheet, then move the `background-image` style to an inline `style=` attribute on the actual element